docs/core/tools/dotnet-run.md

```diff
@@ -81,6 +85,10 @@ The name of the launch profile (if any) to use when launching the application. L
 
 Doesn't build the project before running.
 
+`--no-dependencies`
+
+Set this flag to ignore project to project references and only restore the root project.
```

Comment:
Same here.

Set this flag to ignore ... and only restore -> Ignores ... and only restores

project to project -> project-to-project